src/backend/kvm/thread.rs

```diff
@@ -46,10 +47,7 @@ impl<P: KeepPersonality + 'static> super::super::Keep for RwLock<super::Keep<P>>
 }
 
 impl<P: KeepPersonality> Thread<P> {
-    pub fn balloon(&mut self, req: &Request) -> Result<[Register<usize>; 2], i32> {
-        let log2: usize = req.arg[0].into();
-        let npgs: usize = req.arg[1].into(); // Number of Pages
-        let addr: usize = req.arg[2].into(); // Guest Physical Address
+    pub fn balloon(&mut self, log2: usize, npgs: usize, addr: usize) -> Result<usize, i32> {
```

Comment:
It could be a bit cleaner to use `sallyport::Result` maybe?

Reply:
done